src/desktop/apps/rss/test/routes.test.js

```diff
@@ -167,5 +173,12 @@ describe("Routes", () => {
         "https://api.instagram.com/oembed?url=https://www.instagram.com/p/Bh-Az5_gaVB/?taken-by=artsy"
       )
     })
+
+    it("This case is about fetching the undefined data", async () => {
```

Comment:
For future reference, we typically use the "it" function call as part of the test name - so this test name would read better as `it("fetches the undefined data")`, or "it fetches the undefined data" as we scan it.